plugin/pulp/plugin/tasking.py

```diff
@@ -1,26 +1,24 @@
-from pulp.app.models import Task
+from pulp.app import models
 from pulp.exceptions import exception_to_dict
 from pulp.tasking import get_current_task_id
 
 
-class TaskController(object):
+class Task(object):
     """
-    An interface to the currently running task.
+    The task which is currently executing.
```

Comment:
This probably needs a sentence about this being a facade. Something similar to what you say in your PR description.